README.md

```diff
@@ -23,60 +22,57 @@ mechanics and functionality.
 It is completely open source; community involvement is highly encouraged.
 
 If you wish to contribute ideas or code please visit our site linked below or
-make pull requests to our [Github repository](https://github.com/TrinityCore/TrinityCore).
+make pull requests to our [Github repository](https://github.com/TrinityCore/TrinityCore/pulls).
 
 For further information on the TrinityCore project, please visit our project
-website at [TrinityCore.org](http://www.trinitycore.org).
+website at [TrinityCore.org](https://www.trinitycore.org).
 
 ## Requirements
 
 
-Software requirements are available in the [wiki](http://www.trinitycore.info/display/tc/Requirements) for
+Software requirements are available in the [wiki](https://trinitycore.atlassian.net/wiki/display/tc/Requirements) for
```

Comment:
I will change it to the https://www.trinitycore.info counterpart - the rest is ok?